examples/kubernetes/snapshot/README.md

```diff
@@ -10,7 +10,8 @@ This driver implements basic volume snapshotting functionality using the [extern
 
 1. The `VolumeSnapshotDataSource` must be set in `--feature-gates=` in the `kube-apiserver`.
 
-1. The [aws-ebs-csi-driver driver](https://github.com/kubernetes-sigs/aws-ebs-csi-driver) is installed.
+1. The [aws-ebs-csi-driver driver](https://github.com/kubernetes-sigs/aws-ebs-csi-driver) with alpha features is installed.
+1. If your cluster does not come pre-installed with Snapshot Beta CRDs, install the [Snapshot Beta CRDs](https://github.com/kubernetes-csi/external-snapshotter/tree/master/client/config/crd)
```

Comment:
I think it'd be more useful to link directly to installation part of their README: https://github.com/kubernetes-csi/external-snapshotter#usage

Reply:
Thanks @ayberk , I agree on this especially should any further changes be made on where their files are stored in the future it would be better to link directly to their README. Please see the new commit.